### PR TITLE
Python3 support for docker containers

### PR DESCRIPTION
**Pull request by engapa**
Adding python 3 compatibility for docker containers

### CODE REVIEW

**Comment by engapa** [comment]
With this commit I want to get python 3 compatibility, and older versions of spark as well (i.e spark 2.2.0)

**Comment by glorysdj** [comment]
I will test this PR in our internal PR validation job.

File: docker/install-python.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+${PIP} install --no-binary pandas -I pandas
+${PIP} install scikit-learn matplotlib seaborn jupyter wordcloud moviepy requests h5py opencv-python tensorflow
+
+if [[ "$PYTHON" != "python3" ]]; then
+    ${PYTHON} -m pip install -U ipykernel
+    ${PYTHON} -m ipykernel install --user
+else
+    ln -s /usr/bin/python3 /usr/bin/python
+    ln -s /usr/bin/pip3 /usr/local/bin/pip
+    #Fix tornado await process
```
[the remark this answers is not in the document]
Comment (engapa) [reply docker/install-python.sh]: There are some "call" to python and the executable file in path is python3, so the fast way to fix that is add this link. Keep on mind that one container, til now, supports only one python version.

File: docker/Dockerfile [hunk 2]
```diff
@@ -43,25 +44,21 @@ ENV PATH                                ${JAVA_HOME}/bin:${PATH}
...
-    apt-get install -y build-essential python python-setuptools python-dev && \
-    wget https://bootstrap.pypa.io/get-pip.py && \
-    python2 get-pip.py && \
-    pip2 install --upgrade setuptools && \
-    pip2 install numpy scipy && \
-    pip2 install --no-binary pandas -I pandas && \
-    pip2 install scikit-learn matplotlib seaborn jupyter wordcloud moviepy requests h5py opencv-python tensorflow && \
-    python2 -m pip install ipykernel && \
-    python2 -m ipykernel install --user
+ADD ./install-python.sh ./
```
[the remark this answers is not in the document]
Comment (engapa) [reply docker/Dockerfile]: Do you mean you want to change relative path (WORKDIR) for absolute paths (/opt/work/) for all scripts added to the container ?

[the remark this answers is not in the document]
Comment (engapa) [reply docker/Dockerfile]: I guess that the answer to above question is YES, so ... Dockerfile comes with those changes

**Comment by glorysdj** [comment]
jenkins passed 
http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Docker/3/console

**Pull request merged**